socorro/scripts/__init__.py

```diff
@@ -75,6 +76,66 @@ def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, value)
 
 
+class FallbackToPipeAction(argparse.Action):
```

Comment:
There's a test for `FlagAction` here:

https://github.com/mozilla-services/socorro/blob/master/socorro/unittest/scripts/test_init.py#L12

It's probably worth adding a rough test for `FallbackToPipeAction`.